pages/trips/[id]/packing-list/index.js

```diff
@@ -0,0 +1,88 @@
+import StyledHeader from "../../../../components/StyledHeader";
+import { useRouter } from "next/router";
+import { useNewTripStore } from "../../../../stores/useNewTripStore";
+import StyledListContainer from "../../../../components/StyledListContainer";
```

Comment:
This is pretty minor but.

I would have the imports from external packages up top.
And then I would group the component imports together.

Reply:
Thank you @onemanwenttomow for your review! I just changed it following your advice.